doc/source/ray-observability/key-concepts.rst

```diff
@@ -75,11 +75,12 @@ Profiling
 ---------
 Profiling is way of analyzing the performance of an application by sampling the resource usage of it. Ray supports various profiling tools:
 
-- CPU profiling for Worker processes, including integration with :ref:`py-spy <dashboard-profiling>` and :ref:`cProfile <dashboard-cprofile>`
-- Memory profiling for Worker processes with :ref:`memray <memray-profiling>`
+- CPU profiling for Driver and Worker processes, including integration with :ref:`py-spy <dashboard-profiling>` and :ref:`cProfile <dashboard-cprofile>`
+- Memory profiling for Driver and Worker processes with :ref:`memray <memray-profiling>`
 - Built in Task and Actor profiling tool called :ref:`ray timeline <ray-core-timeline>`
+- GPU profiling with `Pytorch Profiler`_
```

Comment:
Is this linked to our profiling.md page?

Reply:
Updated. It is now